docs/guides/getting-started/prerequisites.md

```diff
@@ -47,6 +47,17 @@ Alternatively, you could use `winget` to install rustup using the following comm
 winget install --id Rustlang.Rustup
 ```
 
+:::caution MSVC toolchain as default
+
+For full support for Tauri and tools like [`trunk`] make sure the MSVC Rust toolchain is the selected `default host triple` in the installer dialog.<br/>Depending on your system it should be either `x86_64-pc-windows-msvc`, `i686-pc-windows-msvc`, or `aarch64-pc-windows-msvc`.
```

Comment:
Is trunk supposed to be a link here? If not then remove the brackets around it. I'd also remove the `<br/>`, I think it's fine to be one paragraph.

Reply:
Yeah it was supposed to be a link but i forgot to add that to my review comment...